### PR TITLE
python-selenium

### PR DESCRIPTION
**Pull request by nacho00112**
Finally a termux package for Selenium, i taked the idea from the issue i opened at [issue#375](https://github.com/termux-user-repository/tur/issues/375), this pull-request add the following packages: python-selenium and, python-seledroid and seledroid-app because they are dependencies, seledroid-app uses termux-open to install the application that allows using the phone's chromium via python-seledroid, seledroid-app temporarily enables allow-external-apps in ~/.termux/termux.properties and then install the application, once it is installed, the allow-external-apps comes back to his original configuration, python-selenium imports the things from python-seledroid, this is the first pull-request of repository that i have made, and the first termux package i have created too, so if i did something wrong please let me know so i can solve it.

### CODE REVIEW

**Comment by SunPodder** [comment]
If you're naming one of your packages `python-selenium` then it must be installing only the core selenium package independently from seledroid. Please look into that.

**Comment by nacho00112** [comment]
Is a dummy selenium, so we can import via `selenium` instead of `seledroid`

**Comment by SunPodder** [comment]
Then this name might be perfect `python-selenium-is-seledroid`. `python-selednium` will create misconception among the users as most of them will assume its the selenium package, not seledroid

**Comment by licy183** [comment]
It seems that seledroid is unmaintained, and I think the original selenium will work on Termux because TUR has webdriver in chromium package. If you really need this package, please just name it `python-seledroid`.

**Comment by nacho00112** [comment]
> It seems that seledroid is unmaintained, and I think the original selenium will work on Termux because TUR has webdriver in chromium package. If you really need this package, please just name it `python-seledroid`.

We can see it as a final version

> It seems that seledroid is unmaintained, and I think the original selenium will work on Termux because TUR has webdriver in chromium package. If you really need this package, please just name it `python-seledroid`.

Since luanon used all his knowledge we can see it as a final version, anyway for now i think is no need to edit the code, until we encounter some bug, but it is quite stable, i didn't find any bug

**Comment by nacho00112** [comment]
Seems the i686 CI check depends on a non-existing package "sse2neon", the other checks builds correctly

File: tur/seledroid-app/build.sh
Comment (licy183) [inline tur/seledroid-app/build.sh]: I think these code should be store in a seperate file and copy this file to the target folder in `build.sh`. It is not readable for maintainers.

File: tur/python-selenium-is-seledroid/build.sh
Comment (licy183) [inline tur/python-selenium-is-seledroid/build.sh]: The same here.

**Review by licy183 — CHANGES_REQUESTED** [review]

File: tur/seledroid-app/build.sh
Comment (nacho00112) [reply tur/seledroid-app/build.sh]: The thing is that i don't know how to do that, i believed that it is not possible, but now i learned how, i already pushed the commit with the changes

File: tur/python-selenium-is-seledroid/build.sh
Comment (nacho00112) [reply tur/python-selenium-is-seledroid/build.sh]: Here i changed too

**Comment by nacho00112** [comment]
Everything should work stable now

**Comment by T-Dynamos** [comment]
@nacho00112 You should consider squashing commits.

**Comment by nacho00112** [comment]
I tried and googled a lot but i can't merge the commits into one

**Pull request closed without merging**
